prime-router/metadata/fhir_mapping/hl7/datatypes/XTN/XTNExtension.yml

```diff
@@ -0,0 +1,80 @@
+url:
+  type: SYSTEM_URL
+  value: xtn-contact-point
+
+extension:
+  generateList: true
+  expressionType: nested
+  expressions:
+    - condition: $telNum NOT_NULL
+      generateList: true
+      expressionType: nested
+      vars:
+        telNum: String, XTN.1
```

Comment:
nit: maybe change the var names in this file to match the patterns of the others `xtn1`, `xtn2`, etc

Reply:
yep will do